### ISSUE TITLE
No longer works in Node.js 0.8 in a patch version update

### ISSUE THREAD

**Issue by dougwilson** [petkaantonov/bluebird#602]
```
$ node -pe "require('bluebird')"

node_modules\bluebird\js\main\async.js:77
        if (!descriptor.configurable) {
                       ^
TypeError: Cannot read property 'configurable' of undefined
    at Object.<anonymous> (node_modules\bluebird\js\main\async.js:77:24)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
    at module.exports (node_modules\bluebird\js\main\promise.js:13:13)
    at Object.<anonymous> (node_modules\bluebird\js\main\bluebird.js:9:39)
    at Module._compile (module.js:449:26)
```
There is no problem at all with 2.9.23...

**Comment by petkaantonov** [comment]
Fixed in 2.9.25 but you really should update :scream:

**Comment by dougwilson** [comment]
> you really should update

I cannot agree with you more T_T

> Fixed in 2.9.25

Thank you so much!
